data/korath/korath missions.txt

```diff
@@ -257,3 +257,588 @@ mission "Small Scale Delivery"
+
+			label done
+			`	"Your payment is <payment>, a bit higher than promised. Thank you for the prompt delivery, Captain <last>!"`
+	on visit
+		dialog "You've arrived at <planet> but you left some of the fish behind. You should take off and wait for your escorts to arrive."
+
+
+
+mission "Korath: Assistance Warning"
+	assisting
+	invisible
+	source
+		government Korath
+	to offer
+		has "Korath [F02]: A Fishy Proposition: offered"
+	on offer
+		conversation
+			`Just after the ship departs, you remember the warning from Iris: openly displaying your association with the Korath will anger half the galaxy. Hopefully that was just hyperbole...`
+				decline
+
+
+
+mission "Korath [F03]: Rabasot Pa'ah Failed Landings"
+	name "Land on <planet>."
+	description "Try again to land on <planet> after your last attempt failed."
+	landing
+	repeat 3
+	source "Rabasot Pa'ah"
+	destination "Rabasot Pa'ah"
+	on offer
+		conversation
+			branch "first attempt"
+				"Korath [F03]: Rabasot Pa'ah Failed Landings: offered" == 0
+			branch "second attempt"
+				"Korath [F03]: Rabasot Pa'ah Failed Landings: offered" == 1
+			branch "third attempt"
+
+			label "first attempt"
+			scene scene/purple-lightning
+			action
+				log "Failed to land on Rabasot Pa'ah due to a powerful atmospheric event brought on by unusual interactions between the planet's atmosphere and the nova it orbits. Reconfiguring the shields to act as a plow got you out of the storm. That shield configuration should allow safe transit through the atmosphere in your next landing attempt."
+			`As you descend through the magnetosphere, your computer warns you of a build-up of unusual charged particles, muons, which is a bit odd. From your training on hyperspace exit and entrance, you recall that the mean lifetime of a muon is measured in microseconds. Something here must be actively generating them.`
+			`	You travel deeper into the atmosphere, approaching what appears to be an inactive volcanic island chain. As usual, your ship's descent causes air currents, but the strange atmosphere turns those currents into a powerful storm. The muons mix with this event, creating catastrophic explosions of electrical discharge.`
+			`	Recalling what you've read about extreme atmospheric landings, you reconfigure your shields to act as a plow, pushing through the storm. At first, it isn't enough, so you pull upward, hoping the lower atmospheric density will let you escape. Once you're free, you realize you've already reached the exosphere.`
+			`	A quick analysis by your ship's computer tells you the shield plow technique should get you all the way down to the surface, if you avoid eddies in the magnetosphere. It seems those eddies interact with the nova in an odd manner that produces vast amounts of muons, which ride your ship through the atmosphere.`
```

Comment:
I feel like 'ride' should be changed, unless it's just a meaning of the word which I haven't seen before. What does it mean for the muons to 'ride' the ship through the atmosphere? I imaging the intent is that the muons are essentially creating lightning storms, and the combination of lightning and extreme turbulence is the danger? So how about;
`...vast amounts of muons, which create an intense storm centered on your ship.`

Reply:
I tweaked the explanation to be more consistent. How is it now?